js/client/modules/@arangodb/test-helper.js

```diff
@@ -471,6 +472,16 @@ exports.getEndpointsByType = function (type) {
     .map(endpointToURL);
 };
 
+exports.triggerMetrics = function () {
+  let coordinators = exports.getEndpointsByType("coordinator");
+  exports.getRawMetric(coordinators[0], '?mode=write_global');
+  for (let i = 1; i < coordinators.length; i++) {
+    let c = coordinators[i];
+    exports.getRawMetric(c, '?mode=trigger_global');
+  }
+  require("internal").sleep(2);
```

Comment:
why 2 seconds?

Reply:
This code was written by @MBkkt. I just put implementation for server and for client in corresponding files.

Reply:
We need to wait a little bit unless we can be able to obtain metrics about inverted indexes on cluster deployment